projects/transactions-journey/src/lib/transactions-journey-config.service.ts

```diff
@@ -1,28 +1,6 @@
-import { Inject, Injectable, InjectionToken, Optional } from '@angular/core';
-
-export interface TransactionsJourneyConfiguration {
-  pageSize: number;
-}
-
-export const TransactionsJourneyConfigurationToken = new InjectionToken<TransactionsJourneyConfiguration>('TransactionsJourneyConfiguration injection token');
-
-const configDefaults: TransactionsJourneyConfiguration = {
-  pageSize: 20,
-};
+import { Injectable } from '@angular/core';
 
 @Injectable()
-export class TransactionsJourneyConfigService {
-  private config: TransactionsJourneyConfiguration;
-
-  constructor(@Optional() @Inject(TransactionsJourneyConfigurationToken) config: TransactionsJourneyConfiguration ) {
-    this.config = { ...configDefaults, ...config };
-  }
-
-  get defaults(): TransactionsJourneyConfiguration {
-    return configDefaults;
-  }
-
-  get pageSize(): number {
-    return this.config.pageSize;
-  }
+export class TransactionsJourneyConfiguration {
```

Comment:
👍  I like this refactoring. What was the reason of refactoring? Was the previous implementation not working with Angular 12?

Reply:
I just wanted to make the code more concise! It was working ok even without it.
Journey authors liked the simplification too.